js/views/search/Suggestions.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jquery';
+import loadTemplate from '../../utils/loadTemplate';
+import BaseView from '../baseVw';
+
+export default class extends BaseView {
+  constructor(options = {}) {
+    const opts = {
+      initialState: {
+        suggestions: [],
+        ...options.initialState || {},
+      },
+      ...options,
+    };
+
+    super(opts);
+    this.options = opts;
+  }
+
+  className() {
+    return 'suggestions flex gutterH row tx5 noOverflow';
+  }
+
+  events() {
+    return {
+      'click .js-suggestion': 'onClickSuggestion',
+    };
+  }
+
+  onClickSuggestion(e) {
+    const suggestion = $(e.target).data('suggestion');
+    this.trigger('clickSuggestion', { suggestion });
+  }
+
+  render() {
+    super.render();
+    const state = this.getState();
+    loadTemplate('search/Suggestions.html', t => {
+      this.$el.html(t({
+        ...this.options,
```

Comment:
Is it necessary to pass the options in here? Sometimes you want one or two to get in, but it seems safer and cleaner to explicitly pass in what you need instead of completely passing in the view options to the template.

Reply:
Oops. Leftover from when I passed the suggestions in over options. Thanks!